src/startup_checks.cpp

```diff
@@ -135,7 +135,9 @@ bool AS::Joystick::check_controller_type(ros::NodeHandle * nodeH)
     else if (controller_string == "LOGITECH_G29")
     {
       PublishControl::controller = LOGITECH_G29;
-
+			
+			PublishControl::max_rot_rad = 7.85;
+			
```

Comment:
Can you please fix the spacing here? It's not a huge deal but we should keep with the standards (4 spaces).